### ISSUE TITLE
[Programmatic Access - Logic apps - Parameters Tab]: "Headers" and "Queries" labels are not associated with the edit fields.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6097]
### Describe the issue 

User Experience: 

 User who are dependent on screen reader won't get enough information about the control as screen reader doesn't provide information of label associated with radio buttons.

 Note: User credentials should NOT be included in the bug.

Pre-requisites:

- Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed. 
- Navigate to Azure control present on the right side and select it. 
- Navigate to LogicApp icon present under workspace section, click on tat and select 'Create New Project'. 
- Navigate select stateful workflow, then enter the name of the workflow. 
- Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter. 
- Navigate to 'Open in current window' and select it. 
- A dialog will appear, select 'Yes', now the workflow will open in new window. 

Repro Steps:

- Open Visual Studio app. 
- Navigate to any workflow and right click on it, then select 'open designer'. 
- Navigate all the controls of the designer screen. 
- Now add "HTTP" action and select it from designer screen after adding it. 
- New dialog appears, navigate all the controls of 'Parameters' tab. 
- Verify if "Headers" and "Queries" labels are not associated with the edit fields. 

Actual Result: 

 "Headers" and "Queries" labels are not associated with the edit fields present under "Parameters" tab. Screen reader JAWS only announces as "Enter key edit".

Expected result: 

 "Headers" and "Queries" labels should be associated with the edit fields present under "Parameters" tab. Screen reader JAWS should announce as "Headers Enter key edit" and "Queries Enter key edit".

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
@Eric-B-Wu could you take a look at this one?

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29374877
